### DISCUSSION TITLE
Particle Boron - Add a sample that uses the builtin LTE modem

### DISCUSSION THREAD

**Discussion by zpm1066** [zephyrproject-rtos/zephyr#69521] [General]
**Is your enhancement proposal related to a problem? Please describe.**
I'm trying to use the LTE modem on the Particle Boron. The Boron seems to be pretty well supported board for quite sometime and was wondering if someone can share an example to get things off the ground for anyone new to using the LTE modem.

**Describe the solution you'd like**
I'd like to see a modem sample added for Particle Boron board.

**Describe alternatives you've considered**
I've been using the Particle DeviceOS that fully supports the Boron LTE modem and connects to their cloud. However, I'd like to run Zephyr on the Boron board and use it to access other cloud services.

**Additional context**

**Comment by WilliamGFish** [comment]
The modem on the Boron is a UBlox Sara (U2, R4 & R5) modem, have a look at the driver for supporting protocols.
Billy..
PS be aware they do like to change the firmware so it automatically connects to the particle backend

**Comment by zpm1066** [comment]
@WilliamGFish Thanks. Will do.
I have a mixture of the two Sara modem versions.
Will I be able to connect to the Particle cloud once I put Zephyr on the Boron and still use their free tier for LTE access?

**Comment by WilliamGFish** [comment]
> @WilliamGFish Thanks. Will do.
> I have a mixture of the two Sara modem versions.
> Will I be able to connect to the Particle cloud once I put Zephyr on the Boron and still use their free tier for LTE access?
> 
> 

Two different questions:
- Can zephyr connect and control the Sara modems? Absolutely.
- Can you connect to the particle platform using zephyr? Now that's a question for particle.
Billy..

**Comment by zpm1066** [comment]
Are you using the Boron modem? If so, Are you connecting to a cloud service like Golioth, AWS, Nordic or other? 

I’ll check with Particle but doubt very much they’ll support a Boron running Zephyr connecting to their cloud.

**Reply by zpm1066** [reply]
@WilliamGFish  
As expected, unfortunately connection to the Particle Cloud with a Boron running Zephyr is a no go according to Particle.

I assume , I'd need to send AT commands over the UART serial port for cellular connection? 

What cloud service are you using with the Boron?

Anyway, it looks like I need to get a handle on the cellular modem driver first.
